notebooks/community/managed_notebooks/Subscriber_Churn_Prediction/Subscriber-Churn-notebook.ipynb

```diff
@@ -0,0 +1,1449 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    # Undeploy the model resource
</pre><p> </p><p>Use Vertex SDK for Python, not gcloud for cleanup.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/99/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='105'/>